lib/features/support/option.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Provides the set of Maze Runner command line options
+class Option
```

Comment:
Should this be in the `Maze` module?

Reply:
Yes, I realised that after creating the PR - it's fixed on my next branch.